services/wallet/community/manager.go

```diff
@@ -0,0 +1,73 @@
+package community
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/status-im/status-go/services/wallet/thirdparty"
+)
+
+const failedCommunityFetchRetryDelay = 1 * time.Hour
+
+type Manager struct {
+	db *DataDB
+}
+
+func NewManager(db *sql.DB) *Manager {
```

Comment:
makes more sense to me to have the `Manager` hold the instance to the `CommunityInfoProvider`, I don't think that's going to change depending on who calls `FetchCommunityInfo`

Reply:
yeap, I'm OK with this. Just a small comment below